GsaGH/Parameters/2_Geometry/GsaModelDefaultValues.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GsaGH.Parameters {
+
+  public struct GsaMemberDefaults {
```

Comment:
@DominikaLos @psarras I feel this is not required. Just look for wrapper class "public class GSAElement" and do all default initilization there else you will endup making changes at multiple places and even for load panel